bin/docserv

```diff
@@ -284,6 +296,7 @@ class DocConfParser:
             return False
         target = self.build_instruction['target']
         xml_root = self.tree.getroot()
+        list_deliverables = []
         for xml_deliverable in xml_root.findall(".//productindex[@id='%s']/docset[@id='%s']/builddocs/language[@code='%s']/deliverable" % (self.build_instruction['product'], self.build_instruction['docset'], self.build_instruction['lang'])):
```

Comment:
Argh, `.//` is one of the worst XPath expressions. It traverse the **complete** tree, which can be very time consuming. Do you really need it? Are the `productindex` elements are really that deeply nested? (not sure, as I don't have the XML structure in mind)

Maybe you can rethink this ugly XPath to use a better expression.

Reply:
True, "/docservconfig/productindex/" should work as well.